app/scripts/fetch_xulrunner

```diff
@@ -131,13 +131,15 @@ function modify_omni {
 	replace_line 'MOZ_MAINTENANCE_SERVICE:' 'MOZ_MAINTENANCE_SERVICE: false \&\&' modules/AppConstants.jsm
 	
 	# Prompt if major update is available instead of installing automatically on restart
-	replace_line 'if \(!updateAuto\) \{' 'if (update.type == "major") {
-      LOG("UpdateService:_selectAndInstallUpdate - prompting because it is a major update");
-      AUSTLMY.pingCheckCode(this._pingSuffix, AUSTLMY.CHK_SHOWPROMPT_PREF);
-      Services.obs.notifyObservers(update, "update-available", "show-prompt");
-      return;
-    }
-    if (!updateAuto) {' modules/UpdateService.jsm
+	if [[ "$1" != "aarch64" ]]; then
```

Comment:
Why are you disabling this? UpdateService.jsm will exist even if we haven't gotten the updater working yet.

Reply:
I get this:
```
grep: modules/UpdateService.jsm: No such file or directory
if \(!updateAuto\) \{ not found in modules/UpdateService.jsm -- aborting
```
Maybe this has to do with this specific Debian Firefox...

Reply:
Don't know if that helps, but these are the changes. `firefox-102.15.1esr` is the x86_64 release from Mozilla and `firefox_deb` is the content of the `.deb`. I unzipped `omni.ja` in both directories. To me this doesn't look like more changes than the ones of the uploader (see at the very end of the diff).

```diff
$ diff -bur --color firefox-102.15.1esr/firefox firefox_deb/usr/lib/firefox-esr/
diff -bur --color firefox-102.15.1esr/firefox/application.ini firefox_deb/usr/lib/firefox-esr/application.ini
--- firefox-102.15.1esr/firefox/application.ini	2023-09-12 04:31:32.000000000 +0200
+++ firefox_deb/usr/lib/firefox-esr/application.ini	2023-08-29 22:40:53.000000000 +0200
@@ -4,23 +4,21 @@
 [App]
 Vendor=Mozilla
 Name=Firefox
-RemotingName=firefox-esr
-Version=102.15.1
-BuildID=20230912014407
+RemotingName=Firefox-esr
+Version=102.15.0
+BuildID=20230822151617
 SourceRepository=https://hg.mozilla.org/releases/mozilla-esr102
-SourceStamp=e6d531e487447f2d6a3a902c146b0909029996c6
+SourceStamp=6db0d7b1b1653e0bbe3058e373235c673205adb4
 ID={ec8030f7-c20a-464f-9b0e-13a3a9e97384}

 [Gecko]
-MinVersion=102.15.1
-MaxVersion=102.15.1
+MinVersion=102.15.0
+MaxVersion=102.15.0

 [XRE]
 EnableProfileMigrator=1

 [Crash Reporter]
 Enabled=1
-ServerURL=https://crash-reports.mozilla.com/submit?id={ec8030f7-c20a-464f-9b0e-13a3a9e97384}&version=102.15.1&buildid=20230912014407
+ServerURL=https://crash-reports.mozilla.com/submit?id={ec8030f7-c20a-464f-9b0e-13a3a9e97384}&version=102.15.0&buildid=20230822151617

-[AppUpdate]
-URL=https://aus5.mozilla.org/update/6/%PRODUCT%/%VERSION%/%BUILD_ID%/%BUILD_TARGET%/%LOCALE%/%CHANNEL%/%OS_VERSION%/%SYSTEM_CAPABILITIES%/%DISTRIBUTION%/%DISTRIBUTION_VERSION%/update.xml
Only in firefox_deb/usr/lib/firefox-esr/browser: defaults
Binary files firefox-102.15.1esr/firefox/browser/features/formautofill@mozilla.org.xpi and firefox_deb/usr/lib/firefox-esr/browser/features/formautofill@mozilla.org.xpi differ
Binary files firefox-102.15.1esr/firefox/browser/features/pictureinpicture@mozilla.org.xpi and firefox_deb/usr/lib/firefox-esr/browser/features/pictureinpicture@mozilla.org.xpi differ
Binary files firefox-102.15.1esr/firefox/browser/features/webcompat@mozilla.org.xpi and firefox_deb/usr/lib/firefox-esr/browser/features/webcompat@mozilla.org.xpi differ
Binary files firefox-102.15.1esr/firefox/browser/features/webcompat-reporter@mozilla.org.xpi and firefox_deb/usr/lib/firefox-esr/browser/features/webcompat-reporter@mozilla.org.xpi differ
Binary files firefox-102.15.1esr/firefox/browser/omni.ja and firefox_deb/usr/lib/firefox-esr/browser/omni.ja differ
diff -bur --color firefox-102.15.1esr/firefox/chrome/toolkit/content/global/aboutSupport.xhtml firefox_deb/usr/lib/firefox-esr/chrome/toolkit/content/global/aboutSupport.xhtml
--- firefox-102.15.1esr/firefox/chrome/toolkit/content/global/aboutSupport.xhtml	2010-01-01 00:00:00.000000000 +0100
+++ firefox_deb/usr/lib/firefox-esr/chrome/toolkit/content/global/aboutSupport.xhtml	2010-01-01 00:00:00.000000000 +0100
@@ -85,28 +85,7 @@
             <td id="distributionid-box"></td>
           </tr>

-          <tr id="update-dir-row" class="no-copy">
-            <th class="column" data-l10n-id="app-basics-update-dir"/>

-            <td>
-              <button id="update-dir-button" data-l10n-id="show-dir-label"/>
-               <span id="update-dir-box" dir="ltr">
-               </span>
-            </td>
-          </tr>
-
-          <tr id="update-history-row" class="no-copy">
-            <th class="column" data-l10n-id="app-basics-update-history"/>
-
-            <td>
-              <button id="show-update-history-button" data-l10n-id="app-basics-show-update-history"/>
-            </td>
-          </tr>
-
-          <tr>
-            <th class="column" data-l10n-id="app-basics-update-channel"/>
-            <td id="updatechannel-box"></td>
-          </tr>

           <tr>
             <th class="column" data-l10n-id="app-basics-user-agent"/>
diff -bur --color firefox-102.15.1esr/firefox/chrome/toolkit/content/global/buildconfig.html firefox_deb/usr/lib/firefox-esr/chrome/toolkit/content/global/buildconfig.html
--- firefox-102.15.1esr/firefox/chrome/toolkit/content/global/buildconfig.html	2010-01-01 00:00:00.000000000 +0100
+++ firefox_deb/usr/lib/firefox-esr/chrome/toolkit/content/global/buildconfig.html	2010-01-01 00:00:00.000000000 +0100
@@ -14,7 +14,7 @@
       <h1>Build Configuration</h1>
       <p>Please be aware that this page doesn't reflect all the options used to build Firefox.</p>
       <h2>Source</h2>
-      <p>Built from <a href="https://hg.mozilla.org/releases/mozilla-esr102/rev/e6d531e487447f2d6a3a902c146b0909029996c6">https://hg.mozilla.org/releases/mozilla-esr102/rev/e6d531e487447f2d6a3a902c146b0909029996c6</a></p>
+      <p>Built from <a href="https://hg.mozilla.org/releases/mozilla-esr102/rev/6db0d7b1b1653e0bbe3058e373235c673205adb4">https://hg.mozilla.org/releases/mozilla-esr102/rev/6db0d7b1b1653e0bbe3058e373235c673205adb4</a></p>
       <h2>Build platform</h2>
       <table class="build-platform-table">
         <tbody>
@@ -22,7 +22,7 @@
             <th>target</th>
           </tr>
           <tr>
-            <td>x86_64-pc-linux-gnu</td>
+            <td>aarch64-unknown-linux-gnu</td>
           </tr>
         </tbody>
       </table>
@@ -35,24 +35,24 @@
             <th>Compiler flags</th>
           </tr>
           <tr>
-            <td>/builds/worker/fetches/clang/bin/clang --sysroot /builds/worker/fetches/sysroot-x86_64-linux-gnu -std=gnu99</td>
-            <td>14.0.4</td>
-            <td>-fcrash-diagnostics-dir=/builds/worker/artifacts -fPIC -ffunction-sections -fdata-sections -fno-math-errno -pthread -pipe</td>
+            <td>/usr/bin/gcc -std=gnu99</td>
+            <td>10.2.1</td>
+            <td>-O2 -ffile-prefix-map=/build/reproducible-path/firefox-esr-102.15.0esr=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -ffunction-sections -fdata-sections -fno-math-errno -pthread -pipe</td>
           </tr>
           <tr>
-            <td>/builds/worker/fetches/clang/bin/clang++ --sysroot /builds/worker/fetches/sysroot-x86_64-linux-gnu -std=gnu++17</td>
-            <td>14.0.4</td>
-            <td>-Qunused-arguments -Qunused-arguments -Wall -Wbitfield-enum-conversion -Wdeprecated-this-capture -Wempty-body -Wformat-type-confusion -Wignored-qualifiers -Wpointer-arith -Wshadow-field-in-constructor-modified -Wsign-compare -Wtype-limits -Wno-error=tautological-type-limit-compare -Wunreachable-code -Wunreachable-code-return -Wunused-but-set-parameter -Wno-invalid-offsetof -Wclass-varargs -Wempty-init-stmt -Wfloat-overflow-conversion -Wfloat-zero-conversion -Wloop-analysis -Wno-range-loop-analysis -Wc++2a-compat -Wcomma -Wenum-compare-conditional -Wimplicit-fallthrough -Werror=non-literal-null-conversion -Wstring-conversion -Wno-inline-new-delete -Wno-error=deprecated-declarations -Wno-error=array-bounds -Wno-error=backend-plugin -Wno-error=free-nonheap-object -Wno-error=return-std-move -Wno-error=atomic-alignment -Wno-error=deprecated-copy -Wformat -Wformat-security -Wno-gnu-zero-variadic-macro-arguments -Wno-psabi -Wthread-safety -Wno-unknown-warning-option -D_GLIBCXX_USE_CXX11_ABI=0 -fno-sized-deallocation -fno-aligned-new -fcrash-diagnostics-dir=/builds/worker/artifacts -fno-exceptions -fPIC -fno-rtti -ffunction-sections -fdata-sections -fno-exceptions -fno-math-errno -pthread -pipe -gdwarf-4 -O3 -fomit-frame-pointer -funwind-tables</td>
+            <td>/usr/bin/g++ -std=gnu++17</td>
+            <td>10.2.1</td>
+            <td>-Wdate-time -D_FORTIFY_SOURCE=2 -Wall -Wempty-body -Wignored-qualifiers -Wpointer-arith -Wsign-compare -Wtype-limits -Wunreachable-code -Wno-invalid-offsetof -Wc++2a-compat -Wduplicated-cond -Wimplicit-fallthrough -Wno-error=maybe-uninitialized -Wno-error=deprecated-declarations -Wno-error=array-bounds -Wno-error=coverage-mismatch -Wno-error=free-nonheap-object -Wno-multistatement-macros -Wno-error=class-memaccess -Wno-error=deprecated-copy -Wformat -Wformat-overflow=2 -Wno-psabi -fno-sized-deallocation -fno-aligned-new -O2 -ffile-prefix-map=/build/reproducible-path/firefox-esr-102.15.0esr=. -fstack-protector-strong -Wformat -Werror=format-security -fno-exceptions -fPIC -fno-rtti -ffunction-sections -fdata-sections -fno-exceptions -fno-math-errno -pthread -pipe -gdwarf-4 -freorder-blocks -O2 -fomit-frame-pointer -funwind-tables</td>
           </tr>
           <tr>
-            <td>/builds/worker/fetches/rustc/bin/rustc</td>
-            <td>1.60.0</td>
-            <td></td>
+            <td>/usr/bin/rustc</td>
+            <td>1.59.0</td>
+            <td>--remap-path-prefix=/build/reproducible-path/firefox-esr-102.15.0esr=.</td>
           </tr>
         </tbody>
       </table>
       <h2>Configure options</h2>
-      <p>MOZ_AUTOMATION=1 MOZILLA_OFFICIAL=1 --enable-update-channel=esr MOZBUILD_STATE_PATH=/builds/worker/.mozbuild MOZ_FETCHES_DIR=/builds/worker/fetches ENABLE_CLANG_PLUGIN=1 MOZ_STDCXX_COMPAT=1 --enable-profile-use=cross --with-pgo-profile-path=/builds/worker/fetches/merged.profdata --with-pgo-jarlog=/builds/worker/fetches/en-US.log MOZ_LTO=cross MOZ_NO_PIE_COMPAT=1 --enable-js-shell --enable-rust-simd --enable-default-toolkit=cairo-gtk3-x11-wayland --with-mozilla-api-keyfile=/builds/mozilla-desktop-geoloc-api.key --with-google-location-service-api-keyfile=/builds/gls-gapi.data --with-google-safebrowsing-api-keyfile=/builds/sb-gapi.data --enable-official-branding</p>
+      <p>--host=aarch64-linux-gnu --target=aarch64-linux-gnu MOZILLA_OFFICIAL=1 --enable-update-channel=esr MOZBUILD_STATE_PATH=/build/reproducible-path/firefox-esr-102.15.0esr/debian/.mozbuild CC=gcc CXX=g++ --enable-alsa --with-app-name=firefox-esr RUSTFLAGS=--remap-path-prefix=/build/reproducible-path/firefox-esr-102.15.0esr=. --enable-system-ffi --enable-default-toolkit=cairo-gtk3-wayland --with-mozilla-api-keyfile=/build/reproducible-path/firefox-esr-102.15.0esr/debian/mls.key --with-google-location-service-api-keyfile=/build/reproducible-path/firefox-esr-102.15.0esr/debian/google.key --with-google-safebrowsing-api-keyfile=/build/reproducible-path/firefox-esr-102.15.0esr/debian/google.key --with-unsigned-addon-scopes=app,system --with-system-libvpx --without-wasm-sandboxed-libraries --disable-updater MOZ_APP_REMOTINGNAME=Firefox-esr --with-system-libevent --disable-install-strip --with-system-zlib --enable-official-branding --prefix=/usr</p>
     </div>
   </body>
 </html>
Only in firefox-102.15.1esr/firefox/chrome/toolkit/content/mozapps: update
diff -bur --color firefox-102.15.1esr/firefox/components/components.manifest firefox_deb/usr/lib/firefox-esr/components/components.manifest
--- firefox-102.15.1esr/firefox/components/components.manifest	2010-01-01 00:00:00.000000000 +0100
+++ firefox_deb/usr/lib/firefox-esr/components/components.manifest	2010-01-01 00:00:00.000000000 +0100
@@ -1,7 +1,6 @@
 category update-timer nsSearchService @mozilla.org/browser/search-service;1,getService,search-engine-update-timer,browser.search.update.interval,21600
 category update-timer addonManager @mozilla.org/addons/integration;1,getService,addon-background-update-timer,extensions.update.interval,86400
 category addon-provider-module GMPProvider resource://gre/modules/addons/GMPProvider.jsm
-category update-timer nsUpdateService @mozilla.org/updates/update-service;1,getService,background-update-timer,app.update.interval,43200,86400
 category app-startup MainProcessSingleton service,@mozilla.org/main-process-singleton;1 process=main
 component {220cc253-b60f-41f6-b9cf-fdcb325f970f} HandlerService.js
 contract @mozilla.org/uriloader/handler-service;1 {220cc253-b60f-41f6-b9cf-fdcb325f970f} process=main
Binary files firefox-102.15.1esr/firefox/crashreporter and firefox_deb/usr/lib/firefox-esr/crashreporter differ
diff -bur --color firefox-102.15.1esr/firefox/dependentlibs.list firefox_deb/usr/lib/firefox-esr/dependentlibs.list
--- firefox-102.15.1esr/firefox/dependentlibs.list	2023-09-12 05:09:00.000000000 +0200
+++ firefox_deb/usr/lib/firefox-esr/dependentlibs.list	2023-08-29 22:40:53.000000000 +0200
@@ -1,8 +1,8 @@
 libnspr4.so
-libplc4.so
-libplds4.so
 libmozsandbox.so
+libplc4.so
 liblgpllibs.so
+libplds4.so
 libnssutil3.so
 libnss3.so
 libsmime3.so
Only in firefox_deb/usr/lib/firefox-esr/: distribution
Only in firefox-102.15.1esr/firefox: firefox
Binary files firefox-102.15.1esr/firefox/firefox-bin and firefox_deb/usr/lib/firefox-esr/firefox-bin differ
Only in firefox-102.15.1esr/firefox: firefox-bin.sig
Only in firefox_deb/usr/lib/firefox-esr/: firefox-esr
Only in firefox-102.15.1esr/firefox: firefox.sig
Binary files firefox-102.15.1esr/firefox/gmp-clearkey/0.1/libclearkey.so and firefox_deb/usr/lib/firefox-esr/gmp-clearkey/0.1/libclearkey.so differ
Only in firefox-102.15.1esr/firefox/gmp-clearkey/0.1: libclearkey.so.sig
diff -bur --color firefox-102.15.1esr/firefox/greprefs.js firefox_deb/usr/lib/firefox-esr/greprefs.js
--- firefox-102.15.1esr/firefox/greprefs.js	2010-01-01 00:00:00.000000000 +0100
+++ firefox_deb/usr/lib/firefox-esr/greprefs.js	2010-01-01 00:00:00.000000000 +0100
@@ -1307,8 +1307,8 @@
   pref("widget.disable-workspace-management", false);
   pref("widget.titlebar-x11-use-shape-mask", false);
 //@line 3557 "$SRCDIR/modules/libpref/init/all.js"
-//@line 3561 "$SRCDIR/modules/libpref/init/all.js"
-  pref("geo.provider.network.url", "https://www.googleapis.com/geolocation/v1/geolocate?key=%GOOGLE_LOCATION_SERVICE_API_KEY%");
+//@line 3563 "$SRCDIR/modules/libpref/init/all.js"
+  pref("geo.provider.network.url", "https://location.services.mozilla.com/v1/geolocate?key=%MOZILLA_API_KEY%");
 //@line 3566 "$SRCDIR/modules/libpref/init/all.js"
 pref("geo.provider.network.timeToWaitBeforeSending", 5000);
 pref("geo.provider.network.timeout", 60000);
Only in firefox-102.15.1esr/firefox: icons
Binary files firefox-102.15.1esr/firefox/libfreeblpriv3.so and firefox_deb/usr/lib/firefox-esr/libfreeblpriv3.so differ
Binary files firefox-102.15.1esr/firefox/libipcclientcerts.so and firefox_deb/usr/lib/firefox-esr/libipcclientcerts.so differ
Binary files firefox-102.15.1esr/firefox/liblgpllibs.so and firefox_deb/usr/lib/firefox-esr/liblgpllibs.so differ
Binary files firefox-102.15.1esr/firefox/libmozavcodec.so and firefox_deb/usr/lib/firefox-esr/libmozavcodec.so differ
Binary files firefox-102.15.1esr/firefox/libmozavutil.so and firefox_deb/usr/lib/firefox-esr/libmozavutil.so differ
Binary files firefox-102.15.1esr/firefox/libmozgtk.so and firefox_deb/usr/lib/firefox-esr/libmozgtk.so differ
Binary files firefox-102.15.1esr/firefox/libmozsandbox.so and firefox_deb/usr/lib/firefox-esr/libmozsandbox.so differ
Binary files firefox-102.15.1esr/firefox/libmozsqlite3.so and firefox_deb/usr/lib/firefox-esr/libmozsqlite3.so differ
Binary files firefox-102.15.1esr/firefox/libmozwayland.so and firefox_deb/usr/lib/firefox-esr/libmozwayland.so differ
Binary files firefox-102.15.1esr/firefox/libnspr4.so and firefox_deb/usr/lib/firefox-esr/libnspr4.so differ
Binary files firefox-102.15.1esr/firefox/libnss3.so and firefox_deb/usr/lib/firefox-esr/libnss3.so differ
Binary files firefox-102.15.1esr/firefox/libnssckbi.so and firefox_deb/usr/lib/firefox-esr/libnssckbi.so differ
Binary files firefox-102.15.1esr/firefox/libnssutil3.so and firefox_deb/usr/lib/firefox-esr/libnssutil3.so differ
Binary files firefox-102.15.1esr/firefox/libplc4.so and firefox_deb/usr/lib/firefox-esr/libplc4.so differ
Binary files firefox-102.15.1esr/firefox/libplds4.so and firefox_deb/usr/lib/firefox-esr/libplds4.so differ
Binary files firefox-102.15.1esr/firefox/libsmime3.so and firefox_deb/usr/lib/firefox-esr/libsmime3.so differ
Binary files firefox-102.15.1esr/firefox/libsoftokn3.so and firefox_deb/usr/lib/firefox-esr/libsoftokn3.so differ
Binary files firefox-102.15.1esr/firefox/libssl3.so and firefox_deb/usr/lib/firefox-esr/libssl3.so differ
Binary files firefox-102.15.1esr/firefox/libxul.so and firefox_deb/usr/lib/firefox-esr/libxul.so differ
Only in firefox-102.15.1esr/firefox: libxul.so.sig
Only in firefox-102.15.1esr/firefox: META-INF
Binary files firefox-102.15.1esr/firefox/minidump-analyzer and firefox_deb/usr/lib/firefox-esr/minidump-analyzer differ
Only in firefox_deb/usr/lib/firefox-esr/modules: addonsInfo.jsm
diff -bur --color firefox-102.15.1esr/firefox/modules/AppConstants.jsm firefox_deb/usr/lib/firefox-esr/modules/AppConstants.jsm
--- firefox-102.15.1esr/firefox/modules/AppConstants.jsm	2010-01-01 00:00:00.000000000 +0100
+++ firefox_deb/usr/lib/firefox-esr/modules/AppConstants.jsm	2010-01-01 00:00:00.000000000 +0100
@@ -87,8 +87,8 @@
 //@line 119 "$SRCDIR/toolkit/modules/AppConstants.jsm"

   MOZ_UPDATER:
-//@line 122 "$SRCDIR/toolkit/modules/AppConstants.jsm"
-  true,
+//@line 124 "$SRCDIR/toolkit/modules/AppConstants.jsm"
+  false,
 //@line 126 "$SRCDIR/toolkit/modules/AppConstants.jsm"

   MOZ_SWITCHBOARD:
@@ -204,6 +204,10 @@

   get MOZ_UNSIGNED_SCOPES() {
     let result = 0;
+//@line 295 "$SRCDIR/toolkit/modules/AppConstants.jsm"
+    result |= AddonManager.SCOPE_APPLICATION;
+//@line 298 "$SRCDIR/toolkit/modules/AppConstants.jsm"
+    result |= AddonManager.SCOPE_SYSTEM;
 //@line 300 "$SRCDIR/toolkit/modules/AppConstants.jsm"
     return result;
   },
@@ -236,29 +240,29 @@
   DLL_PREFIX: "lib",
   DLL_SUFFIX: ".so",

-  MOZ_APP_NAME: "firefox",
+  MOZ_APP_NAME: "firefox-esr",
   MOZ_APP_BASENAME: "Firefox",
   // N.b.: you almost certainly want brandShortName/brand-short-name:
   // MOZ_APP_DISPLAYNAME should only be used for static user-visible
   // fields (e.g., DLL properties, Mac Bundle name, or similar).
   MOZ_APP_DISPLAYNAME_DO_NOT_USE: "Firefox",
-  MOZ_APP_VERSION: "102.15.1",
-  MOZ_APP_VERSION_DISPLAY: "102.15.1esr",
-  MOZ_BUILDID: "20230912014407",
+  MOZ_APP_VERSION: "102.15.0",
+  MOZ_APP_VERSION_DISPLAY: "102.15.0esr",
+  MOZ_BUILDID: "20230822151617",
   MOZ_BUILD_APP: "browser",
   MOZ_MACBUNDLE_ID: "org.mozilla.firefox",
   MOZ_MACBUNDLE_NAME: "Firefox.app",
   MOZ_UPDATE_CHANNEL: "esr",
   MOZ_WIDGET_TOOLKIT: "gtk",
-  ANDROID_PACKAGE_NAME: "org.mozilla.firefox",
+  ANDROID_PACKAGE_NAME: "org.mozilla.firefox-esr",

   DEBUG_JS_MODULES: "",

   MOZ_BING_API_CLIENTID: "no-bing-api-clientid",
   MOZ_BING_API_KEY: "no-bing-api-key",
-  MOZ_GOOGLE_LOCATION_SERVICE_API_KEY: "AIzaSyB2h2OuRcUgy5N-5hsZqiPW6sH3n_rptiQ",
-  MOZ_GOOGLE_SAFEBROWSING_API_KEY: "AIzaSyC7jsptDS3am4tPx4r3nxis7IMjBc5Dovo",
-  MOZ_MOZILLA_API_KEY: "7e40f68c-7938-4c5d-9f95-e61647c213eb",
+  MOZ_GOOGLE_LOCATION_SERVICE_API_KEY: "AIzaSyD3uzXks34szqk9WhKoFZypVPgdDbT3uPw",
+  MOZ_GOOGLE_SAFEBROWSING_API_KEY: "AIzaSyD3uzXks34szqk9WhKoFZypVPgdDbT3uPw",
+  MOZ_MOZILLA_API_KEY: "91e66841-a83b-487f-9b5d-e460f5225ebf",

   BROWSER_CHROME_URL: "chrome://browser/content/browser.xhtml",

@@ -268,7 +272,7 @@
   // "https://hg.mozilla.org/mozilla-central/rev/6256ec9113c1")
   // On unofficial builds, this is an empty string.
 //@line 375 "$SRCDIR/toolkit/modules/AppConstants.jsm"
-  SOURCE_REVISION_URL: "https://hg.mozilla.org/releases/mozilla-esr102/rev/e6d531e487447f2d6a3a902c146b0909029996c6",
+  SOURCE_REVISION_URL: "https://hg.mozilla.org/releases/mozilla-esr102/rev/6db0d7b1b1653e0bbe3058e373235c673205adb4",

   HAVE_USR_LIB64_DIR:
 //@line 381 "$SRCDIR/toolkit/modules/AppConstants.jsm"
Only in firefox-102.15.1esr/firefox/modules: UpdateListener.jsm
Only in firefox-102.15.1esr/firefox/modules: UpdateService.jsm
Only in firefox-102.15.1esr/firefox/modules: UpdateServiceStub.jsm
Only in firefox-102.15.1esr/firefox/modules: UpdateTelemetry.jsm
Binary files firefox-102.15.1esr/firefox/omni.ja and firefox_deb/usr/lib/firefox-esr/omni.ja differ
Binary files firefox-102.15.1esr/firefox/pingsender and firefox_deb/usr/lib/firefox-esr/pingsender differ
diff -bur --color firefox-102.15.1esr/firefox/platform.ini firefox_deb/usr/lib/firefox-esr/platform.ini
--- firefox-102.15.1esr/firefox/platform.ini	2023-09-12 05:09:00.000000000 +0200
+++ firefox_deb/usr/lib/firefox-esr/platform.ini	2023-08-29 22:40:53.000000000 +0200
@@ -1,5 +1,5 @@
 [Build]
-BuildID=20230912014407
-Milestone=102.15.1
+BuildID=20230822151617
+Milestone=102.15.0
 SourceRepository=https://hg.mozilla.org/releases/mozilla-esr102
-SourceStamp=e6d531e487447f2d6a3a902c146b0909029996c6
+SourceStamp=6db0d7b1b1653e0bbe3058e373235c673205adb4
Binary files firefox-102.15.1esr/firefox/plugin-container and firefox_deb/usr/lib/firefox-esr/plugin-container differ
Only in firefox-102.15.1esr/firefox: plugin-container.sig
Only in firefox-102.15.1esr/firefox: precomplete
Only in firefox-102.15.1esr/firefox: removed-files
Only in firefox-102.15.1esr/firefox: update.locale
Only in firefox-102.15.1esr/firefox: updater
Only in firefox-102.15.1esr/firefox: updater.ini
Only in firefox-102.15.1esr/firefox: update-settings.ini
```